sunshine-testng/src/main/java/io/github/tatools/sunshine/testng/Sunshine.java

```diff
@@ -18,23 +20,7 @@ public static void main(String[] args) {
         if (args != null && args.length > 0) {
```

Comment:
it might better use isBlank() from StringUtils.class

Reply:
@IVKlymenko I don't want to add here additional library for only one check as well as I against any `Utils` or `Helpers` classes in general.